app/ui/src/app/customizations/api-connector/api-connector-list/api-connector-list.component.html

```diff
@@ -53,9 +53,8 @@ <h1>API Client Connectors</h1>
 </div>
 
 <ng-template #confirmDeleteModalTemplate>
-  <delete-confirmation-modal (delete)="onDeleteConfirm($event)">
-    <p class="lead">Removing a connector removes all of it's connections</p>
-    <p>This will remove all connections for this connector. Deleting this connector does not affect running integrations.</p>
-    <p>Are you sure you want to delete the "{{ deletableApiConnectorName }}" connector?</p>
+  <delete-confirmation-modal (delete)="onDeleteConfirm($event)" title="Confirm Delete?">
+    <p class="lead">Are you sure you want to delete the "{{ deletableApiConnectorName }}" connector?</p>
```

Comment:
You can also replace these lines, also you can have arguments in the string templates, for example:

https://github.com/syndesisio/syndesis/blob/master/app/ui/src/app/connections/create-page/configure-fields/configure-fields.component.html#L49